BoxPuzzle script

TosaLipkiss opened the pull request:
script, prefab placeholder box.

Commit:
BoxPuzzle script

script, prefab placeholder box.

Addehman reviewed (approved):
Probably needs to optimized, but works for now.

smeas committed:
Fix formatting

Commit:
Box Pickup - update

Now the BoxPuzzle Prefab finds the playerHand's position

Commit:
Merge branch 'ButtonPuzzle' of https://github.com/MrJackey/Were.Wolf into ButtonPuzzle

Commit:
Button Puzzle - Interaction Update

- Added a check for if the box is hitting a Spike, then it will be repositioned from its starting position;
- Gave the Spike_Obstacle prefab the "Spike"-tag;
- Added an Input named "Interaction" in the Input Manager, set it to have the "F" key on keyboard, and joystick button 2 (X-button) on gamepad.

smeas committed:
Merge branch 'master' into ButtonPuzzle

Commit:
Button Puzzle - Box update

Cleaned the code a bit

Commit:
Button Puzzle Update

clean code

smeas reviewed (approved):
+1, Nice.

MrJackey reviewed (approved):
Looking good
